src/pages/midwife/homeVisit.jsx

```
@@ -32,50 +33,58 @@ const homeVisit = () => {
-
     fetchParentDataForMidwife();
-  }, []);
+    return () => {
+      isComponentMounted = false;
+    };
+  }, [userId, navigate]);
+
+  const isValidLocation =
+    location?.lat &&
+    location?.lng &&
+    !isNaN(location.lat) &&
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)